app/controllers/users.js

```diff
@@ -1,99 +1,104 @@
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable valid-jsdoc */
+
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-  User = mongoose.model('User');
-var avatars = require('./avatars').all();
+import mongoose from 'mongoose';
+import jwt from 'jsonwebtoken';
 
-/**
- * Auth callback
- */
-exports.authCallback = function(req, res, next) {
-  res.redirect('/chooseavatars');
-};
+const User = mongoose.model('User');
+require('dotenv').config({ path: '.env' });
 
 /**
  * Show login form
  */
-exports.signin = function(req, res) {
+exports.signin = function (req, res) {
   if (!req.user) {
     res.redirect('/#!/signin?error=invalid');
   } else {
-    res.redirect('/#!/app');
+    res.redirect('/');
   }
 };
 
 /**
  * Show sign up form
  */
-exports.signup = function(req, res) {
+exports.signup = function (req, res) {
   if (!req.user) {
     res.redirect('/#!/signup');
   } else {
     res.redirect('/#!/app');
   }
 };
 
+exports.loginSuccess = (req, res) => {
+  console.log('Right details');
```

Comment:
Do we really need this `console.log('Right details');` here? If this was used during development then you can take this.